src/meshes/parallel_tree.jl

```diff
@@ -85,11 +85,6 @@ function ParallelTree{NDIMS, RealT}(capacity::Int, center::AbstractArray{RealT},
 
     return t
 end
-function ParallelTree{NDIMS}(capacity::Int, center::AbstractArray{RealT},
-                             length::RealT,
-                             periodicity = true) where {NDIMS, RealT <: Real}
-    ParallelTree{NDIMS, RealT}(capacity, center, length, periodicity)
-end
```

Comment:
Why is this not needed (anymore)?

Reply:
This was only being called from the now changed lines.

Reply:
I see, thanks!